src/modules/RA/DataLayer/components/tabs/PopupTab/index.js

```diff
@@ -50,24 +60,32 @@ const PopupTab = () => {
 
   const updateTemplate = useCallback(debounce(() => {
     if (popupfields.length) {
-      const lines = popupfields.map(({
+      const [titlefield, ...contentfields] = popupfields;
+      const titleLine = (
+        `# {% if ${titlefield.field.name} %}`
+        + `{{${titlefield.field.name}}}`
+        + `{% else %}${titlefield.default}{% endif %}`
+      );
+
+      const lines = contentfields.map(({
         suffix,
         prefix,
         field: { name },
         default: defaultTitle,
       }) => (
-        `${prefix}: {% if ${name}.length > 0} %}{{${name}}} ${suffix}{% else %}${defaultTitle}{% endif %}`
+        `- ${prefix} : {% if ${name} %}{{${name}}} ${suffix}{% else %}${defaultTitle}{% endif %}`
```

Comment:
0 values are defined and shouldn't show default message.

Reply:
Also the prefix is not the line title. the prefix is just before the value. Use the field name instead.

Reply:
Handle value round.

Reply:
Done.